data/brands/amenity/fast_food.json

```
@@ -1927,6 +1927,20 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Jumbo King",
+      "id": "jumboking-aba431",
+      "locationSet": {"include": ["in"]},
+      "tags": {
+        "amenity": "fast_food",
+        "brand": "Jumbo King",
+        "brand:wikidata": "Q6310895",
+        "brand:wikipedia": "en:Jumbo King",
+        "cuisine": "Burger",
```

This `cuisine` tag values are typically written in lowercase..
```suggestion
        "cuisine": "burger",
```